test.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
WARNING: A file should declare new symbols (classes, functions, constants, etc.) and cause no other side effects, or it should execute logic with side effects, but should not do both. The first symbol is defined on line 7 and the first side effect is on line 2.